src/corelibs/U2Core/src/models/GObjectReference.h

```diff
@@ -40,7 +40,7 @@ class U2CORE_EXPORT GObjectReference {
         : docUrl(_docUrl), objName(_objName), entityRef(_entityRef), objType(_type) {
     }
 
-    GObjectReference(const GObject* obj, bool deriveLoadedType = true);
+    explicit GObjectReference(const GObject* obj, bool deriveLoadedType = true);
```

Comment:
I made this explicit because for the constructors in the CreateAnnotationTask it was ambiguous which one is called: with GObject* or GObjectReference as the first parameter.